### PR TITLE
Help Center: Fix Contact form button styling

### PR DESCRIPTION
**Pull request by agrullon95**
Fixes: https://github.com/Automattic/dotcom-forge/issues/8411

## Proposed Changes

* Fixes the button styling when user hovers in disabled and font color when enabled.

Before ( notice hover state when disabled ) :

https://github.com/user-attachments/assets/8354c0c8-c588-4e78-a4d6-233f1b93be07

After: 

https://github.com/user-attachments/assets/a14f1294-9014-444c-915e-8fde1f48fd41

## Why are these changes being made?

* Fix hover state, and remove unnecessary styling.

## Testing Instructions

* Checkout branc or use live link
* Log in with free user
* Open the Help Center, click "Still need help?"
* Hover over the "Continue" button in a disabled state and verify no styling is applied
* Fill out the contact form and hover over the "Continue" button when its enabled to verify the button background gets darker and the font color is still white (you can compare against other similar buttons `/home`)

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

**Review by alshakero — COMMENTED** [review]
What if we remove all the styles except width 100%?

**Comment by agrullon95** [comment]
@alshakero - We can remove the `active` and `hover` styling. 
I would still need additional styling to change the background color to match what it previously was:

CleanShot 2024-07-25 at 13 46 23@2x

**Comment by alshakero** [comment]
This means there is still CSS somewhere else messing with the button. If you delete everything, it should work. It only needs width: 100% and justify-elements: center.

**Review by escapemanuele — CHANGES_REQUESTED** [review]
Removing all the CSS caused this for me, the border disappears when mouse is inside the Help Center :)

image

**Comment by agrullon95** [comment]
@escapemanuele - I fixed the issue.

File: packages/help-center/src/styles.scss [hunk 1]
```diff
@@ -169,7 +169,7 @@ $head-foot-height: 50px;
 			}
 		}
 
-		.components-button {
+		components-button:not(.help-center-contact-form__site-picker-cta) {
```
Comment (arcangelini) [inline packages/help-center/src/styles.scss]:
```suggestion
		.components-button:not(.help-center-contact-form__site-picker-cta) {
```

Does this need to be a class?

**Comment by arcangelini** [comment]
Tested and everything was working as expected.

**Review by arcangelini — APPROVED** [review]
🚀 🚀 🚀 
LGTM

**Pull request merged**